client/actions/lists.js

```diff
@@ -41,3 +41,21 @@ export const showList = (id) => (dispatch) => {
         });
 	});
 };
+
+export const UPDATED_LIST_ITEM = 'UPDATED_LIST_ITEM';
+
+export const updateListItemStatus = (id, status) => (dispatch) => {
+	return fetch(`${backend}/users/me/acclaimed_list/item`, {
```

Comment:
I would have probably put this action creator in a new items.js file, and used a different route, but again since the demo is in an hour and a half, it's fine

Reply:
You're right!